app/views/shared/provider/_header.html.slim

```diff
@@ -28,6 +28,9 @@ header.pf-c-page__header id="user_widget" class='Header' class=('Header--master'
         - if saas?
           li.PopNavigation-listItem
             = fixed_width_icon_link_to "What's new?", 'leaf', '//access.redhat.com/articles/3107441#newfeaturesenhancements', class: 'PopNavigation-link', target: '_blank'
+        - if provider_can_use?(:quick_starts)
+          li.PopNavigation-listItem
+            = fixed_width_icon_link_to 'Quick starts', 'book', provider_admin_quick_starts_path, class: 'PopNavigation-link'
```

Comment:
I thought this should be one word `Quickstarts`.

Reply:
https://en.wiktionary.org/wiki/quickstart

but I don't know, I see it written as people want everywhere

Reply:
Not to Patternfly guys it isn't 😂 https://quickstarts.netlify.app
Anyway, I renamed all our files and code references to quickstart instead of quick_start. The microcopy we leave it to Docs.